modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionStatusException.java

```diff
@@ -15,8 +15,14 @@ public class ObjectDefinitionStatusException extends PortalException {
 	public ObjectDefinitionStatusException() {
 	}
 
-	public ObjectDefinitionStatusException(String msg) {
-		super(msg);
+	public ObjectDefinitionStatusException(String message) {
```

Comment:
@larissacribeiro please run `gw baseline` in the `object-api` folder and commit as `LPS-196918 baseline`, thanks!

Reply:
@carolmariaabb just did, thanks!